superset/assets/tslint.json

```diff
@@ -1,10 +1,11 @@
 {
-    "extends": ["tslint:recommended", "tslint-react"],
-    "jsRules": {
-    },
-    "rules": {
-      "interface-name" : [true, "never-prefix"],
-      "quotemark": [true, "single"]
-    },
-    "rulesDirectory": []
-}
\ No newline at end of file
+  "extends": ["tslint:recommended", "tslint-react"],
+  "jsRules": {},
+  "rules": {
+    "interface-name": [true, "never-prefix"],
+    "quotemark": [true, "single"],
+    "jsx-no-multiline-js": false,
+    "jsx-no-lambda": false
```

Comment:
We don't have these rules enabled in eslint and I found them to be very tedious. Opened https://github.com/apache/incubator-superset/issues/8865 for larger discussion